pdf-service/src/utils/commons.js

```diff
@@ -53,6 +44,47 @@ export const findAndUpdateLocalisation = async (
     defaultTenant
   ).split(".")[0];
 
+
+  let url = egovLocHost + '/localization/messages/v2/_search';
+
+  let request = { 
+    RequestInfo: requestInfo,
+    messageSearchCriteria:{
+      tenantId: statetenantid,
+      locale: locale,
+      codes: []
+    }
+  };
+
+  request.messageSearchCriteria.module = moduleList.toString();
+  request.messageSearchCriteria.codes = codeList.toString().split(",");
+
+  var options = {
+    method: 'POST',
+    body: JSON.stringify(request),
+    headers: {
+        'Content-Type': 'application/json'
+    }
+  }
+
+  let response = await fetch(url, options);
```

Comment:
use `axios`

Reply:
removed unnecessary dependency and use axios for https request call